draft-ietf-rats-corim.md

```diff
@@ -1816,6 +1829,13 @@ compared with the next entry of the Accepted Claims Set array.
 If all entries of the Reference Values array match a corresponding entry in the Accepted Claims Set array, then the `cryptokeys` Reference Value matches.
 Otherwise, `cryptokeys` does not match.
 
+##### Comparison for Integrity Registers {#sec-cmp-integrity-registers}
+
+For each Integrity Register entry in Evidence, the Verifier will use the associated identifier to look up the matching Integrity Register entry in the Reference Value.
```

Comment:
```suggestion
For each Integrity Register entry in Evidence, the Verifier will use the associated identifier to look up the matching Integrity Register entry in the Reference Value. It is expected that the `type` of identifier in the Evidence is same as the one set in the Reference Values, as dictated by the profile.  Otherwise the comparison is undefined behaviour.
```